libs/types/src/cdf/election-results-reporting/convert.ts

```diff
@@ -154,19 +154,24 @@ function getBallotCounts(contest: ResultsReporting.CandidateContest): {
 } {
   const otherCounts = contest.OtherCounts && contest.OtherCounts[0];
 
-  let ballotCount =
-    0 + (otherCounts?.Overvotes || 0) + (otherCounts?.Undervotes || 0);
+  let totalCandidateVotes = 0;
 
   for (const contestSelection of assertDefined(contest.ContestSelection)) {
     const voteCounts = assertDefined(contestSelection.VoteCounts);
     const totalCount = findTotalVoteCounts(voteCounts);
-    ballotCount += totalCount;
+    totalCandidateVotes += totalCount;
   }
 
+  const totalBallots =
+    (totalCandidateVotes +
+      (otherCounts?.Overvotes || 0) +
+      (otherCounts?.Undervotes || 0)) /
+    contest.VotesAllowed;
```

Comment:
Should we validate that these divide evenly? Or do you think it's safe to assume that the math works out?

Reply:
[9498992](https://github.com/votingworks/vxsuite/pull/5344/commits/94989928be322c2f4b7e6a7acf61e875e527db6b)